examples/alert-absence-dedup/README.md

```diff
@@ -7,91 +7,119 @@ them is missing. If one or two timeseries are missing, you want exactly one
 alert. When there is a total outage you still want to get one alert, not 100
 alerts.
 
+The test app generates time series with a custom metric called
+task_latency_distribution with tags based on a partition label. Each
+partition generates its own time series.
+
 The example assumes that you are familiar with Stackdriver Monitoring and
 Alerting. It builds on the discussion in
 [Alerting policies in depth](https://cloud.google.com/monitoring/alerts/concepts-indepth).
 
-## Setup
+## Deploy the app
+
+The example code is based on the Go code in
+[Custom metrics with OpenCensus](https://cloud.google.com/monitoring/custom-metrics/open-census).
+
+[Download](https://golang.org/dl/) and install the latest version of Go.
 
-Create three GCE instances
+Build the test app
 
 ```shell
-ZONE=us-central1-c
-gcloud compute instances create test-instance-1 test-instance-2 test-instance-3 \
-  --zone=$ZONE \
-  --scopes=https://www.googleapis.com/auth/cloud-platform \
-  --boot-disk-size=20GB \
-  --machine-type=f1-micro
+go build
 ```
 
-Edit the file notification_channel.json, replacing the displayName and
-email_address. Create a notification channel with the command
+The instructions here as based on
+[Setting up authentication](https://cloud.google.com/monitoring/docs/reference/libraries#setting_up_authentication)
+for the Stackdriver client library. Note that if you run the test app on a
+Google Cloud Compute Engine instance, Google Kubernetes Engine, or App Engine
+you will not need to create a service account or download the credentials.
+
```

Comment:
What about Cloud Run? You may want to explicitly address it, since the omission may create confusion, since Cloud Run can be run on GKE and the non-GKE version still has access to the Compute Engine metadata service (giving it the appearance of perhaps running on Compute Engine), so it might apply? But, then again, the omission from this list suggests that it doesn't?

Reply:
Added Cloud Run